exercises/practice/isbn-verifier/.meta/example.cairo

```diff
@@ -0,0 +1,64 @@
+pub fn is_valid_isbn(isbn: ByteArray) -> bool {
+    // Step 1: Remove dashes and ensure only valid characters are left.
+    let filtered_isbn = filter(isbn);
+    let size = filtered_isbn.len();
+
+    // Step 2: Check length of valid characters.
+    if size != 10 {
+        return false;
+    }
+
+    // Step 3: Calculate the sum based on the formula.
+    let mut i = 0;
+    let mut sum: u32 = 0;
+    let mut result = true;
```

Comment:
Given how this flag is used, maybe a clearer name would be `let valid = true` - the name conveys the variable's purpose